notebooks/jacques_drivendata_map_floodwater_benchmark_tutorial.ipynb

```diff
@@ -78,8 +78,10 @@
         }
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.    from pytorch_lightning import seed_everything
</pre><p>Nit: The pytorch_lightning seed_everything import should be right below the first import. 😊</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/JayThibs/map-floodwater-satellite-imagery/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>